eventarc/audit-storage/Dockerfile

```diff
@@ -33,9 +33,9 @@ RUN mvn package -DskipTests
 FROM adoptopenjdk/openjdk11:alpine-slim
 
 # Copy the jar to the production image from the builder stage.
-COPY --from=builder /app/target/events-storage-*.jar /events-storage.jar
+COPY --from=builder /app/target/audit-storage-*.jar /audit-storage.jar
```

Comment:
Sorry missed a comment: for this change the pom.xml needs the name filed changed to audit-storage as well

Reply:
Thanks.